#BE-4225 Docs - Enterprise Rework (#600)
appcircleio/appcircle-docusaurus: docs/enterprise-appstore/enterprise-store.md

---
title: Enterprise Store 
description: Learn how to use Enterprise Store in Appcircle
tags: [enterprise app store, enterprise store setup, enterprise store]
sidebar_position: 4
---

> Appcircle has a separate distribution screen designed to make it easy for users to download the shared applications easily.
> 
> For iOS and Android, the users can login from the link shared and then view all the versions shared with them. Downloading iOS and Android binaries are done through the specific flows of each OS.

Reviewer: ```suggestion
Appcircle features a separate distribution screen designed to facilitate easy downloading of shared applications.

For iOS and Android, users can log in via the shared link and view all the versions shared with them. Downloading iOS and Android binaries is managed through the specific flows for each operating system.
```

## Login

When a binary is shared with users by share or notify actions for Live and Beta channels, the users will receive a link for access.

When the link is clicked, users will then be redirected to the Enterprise Store.;

<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-store1.png" />

<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-mobile1.png" />

:::info

Authentication method can be configured from the [store settings](/enterprise-appstore/store-settings#store-authentication).

:::

## Listing and Downloading App version

> Once logged in, users can now see the app version shared from the Enterprise App Store profile. Files can be downloaded with one click.

Reviewer: ```suggestion
Once logged in, users will be able to see the app version shared from the Enterprise App Store profile. Files can be downloaded with a single click.
```
